src/gval/accessors/gval_xarray.py

```diff
@@ -32,6 +33,7 @@ class GVALXarray:
     def __init__(self, xarray_obj):
         self._obj = xarray_obj
         self.data_type = type(xarray_obj)
+        self.benchmark_format = "raster"
```

Comment:
Should the benchmark format be automatically detected and agreement map format be designated?

Reply:
Changed the name to agreement_map_format